ldaptor/scripts/fetchschema.py

```diff
@@ -75,3 +73,7 @@ class MyOptions(
         baseDN=opts["base"], serviceLocationOverrides=opts["service-location"]
     )
     main(cfg)
+
+
+if __name__ == "__main__":
```

Comment:
does it make sense to keep this here?

Reply:
Yeah it's handy for `python -m ldaptor.scripts.fetchschema`

Reply:
I think that this should have a `# pragma: no cover` comment as we will not test the script via this interface.

The script can be tested via the console_script interface.

Reply:
I'm currently prepping smoke tests that use this interface. Calling the function directly is difficult because of the reactor.stop calls